tensorflow/contrib/cmake/external/highwayhash.cmake

```diff
@@ -12,9 +12,9 @@ set(highwayhash_STATIC_LIBRARIES
 )
 set(highwayhash_INCLUDES ${highwayhash_BUILD})
 
-set(highwayhash_HEADERS
-    "${highwayhash_BUILD}/highwayhash/*.h"
-)
+file(GLOB highwayhash_HEADERS RELATIVE
+     "${highwayhash_BUILD}/highwayhash"
+     "${highwayhash_BUILD}/highwayhash/*.h")
```

Comment:
I have no idea why it fails in CI build.
I have logged into the machine it tests and tried running cmake.
It seems to work nicely. I will try rerunning ci build. 
I will also test this on windows.